src/main/java/core/basesyntax/service/impl/ReaderCsv.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Reader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class ReaderCsv implements Reader {
+    private final String filePath;
+
+    public ReaderCsv(String filePath) {
+        this.filePath = filePath;
+    }
+
+    @Override
+    public List<String> readData() {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                lines.add(line);
+            }
+            return DataProcessor.filterTitles(lines);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file");
+        }
+    }
+
+    static class DataProcessor {
+        private static final Set<String> TITLES_TO_REMOVE = Set.of("type,fruit,quantity");
```

Comment:
What if we change the title, for example, we will add whitespace or something? In this case, we will go to the code and add a new value to the set. I suggest just skipping first element of the array to avoid the header